site/content/en/docs/manual/advanced/annotation-with-skeletons/auto-skeletons.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Automatic annotation with Skeletons'
+linkTitle: 'Automatic annotation with Skeletons'
+weight: 2
+description: 'Automatic annotation with Skeletons'
+---
+
+In this guide, we delve into the efficient process of annotating complex
+structures through the implementation of **Skeleton** automatic annotations.
+
+See:
+
+- [Creating auto annotation Skeleton task](#creating-auto-annotation-skeleton-task)
+- [Automatic annotation with Skeletons](#automatic-annotation-with-skeletons)
+  - [Editing skeletons on the canvas](#editing-skeletons-on-the-canvas)
+
+## Creating auto annotation Skeleton task
+
+To auto annotation **Skeleton** task, do the following:
+
+1. Open **Basic configurator**.
+2. On the **Constructor** tab, click **From model**.
+3. From the **Select a model to pick labels** select the model
+   you want to use.
+4. Click on the model's labels, you want to use.
+   Selected labels will become gray.
+
+   ![](/images/auto-annot-sk.jpg)
+
+5. Click **Done**. Labels, that you selected,
+   will appear in the labels window.
+6. (Optional) If you want to adjust labels, within the
+   label, click the **Update** attributes icon.
```

Comment:
P.S. Skeleton points currently may be adjusted only before a task/project created.

Reply:
It means that if to pick a skeleton label from a model in created task/project, user will not be able to adjust them.